index.js

```diff
@@ -299,9 +311,13 @@ module.exports = {
 
         data.status = 'ERROR';
         data.diffPath = diffPath;
-        data.fullDiffPath = path.join(__dirname, diffPath);
-        data.error = `${errorPixelCount} pixels differ - diff: ${diffPath}, img: ${tmpPath}`;
-
+        data.fullDiffPath = diffPath? path.join(__dirname, diffPath) : null;
+        
+        if(reason.diffPath === null){
+          data.error = `Image sizes do not match ${errorPixelCount} pixels differ - img: ${tmpPath}`;
```

Comment:
```suggestion
          data.error = `Image sizes do not match - ${errorPixelCount} pixels differ - img: ${tmpPath}`;
```

Reply:
This is just copying the formatting of the existing error message